vdev/src/platform.rs

```diff
@@ -25,7 +25,7 @@ pub fn data_dir() -> &'static Path {
 pub fn default_target() -> String {
     if cfg!(windows) {
```

Comment:
Nit: could this be `target_os = "windows"` for consistency?

Reply:
Maybe, but we use just `windows` everywhere else. There are two special OS-type configs, `unix` and `windows`. In the case of `unix`, it includes both Linux and macOS (and of course BSDs etc), but I'm not sure if `windows` similarly includes some other sub-targets.